e2e/test/scenarios/permissions/admin-permissions.cy.spec.js

```diff
@@ -734,3 +734,31 @@ describeEE("scenarios > admin > permissions", () => {
     cy.findByText("Sorry, you don't have permission to see this card.");
   });
 });
+
+describe("scenarios > admin > permissions", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("shows permissions help", () => {
+    cy.visit("/admin/permissions");
+    cy.get("main").within(() => {
```

Comment:
Is there a more specific selector we can use here?

Reply:
Good catch, updated where I could